scripts/all-core.sh

```diff
@@ -861,7 +861,7 @@ pre_check_tools () {
         *) set "$@" RUN_ARMCC=0;;
     esac
     # Use a path relative to the currently-sourced file.
-    "$@" "${BASH_SOURCE%/*}"/../../scripts/output_env.sh
+    "$@" "${BASH_SOURCE%/*}"/../../framework/scripts/output_env.sh
```

Comment:
This is correct but I think we can simplify this by removing `../../framework/scripts` now that the two files live in the same directory: `"${BASH_SOURCE%/*}"/output_env.sh` should do the trick.